test/test_torch.py

```diff
@@ -18342,6 +18342,36 @@ def test_atleast(self, device, dtype):
         self._test_atleast_dim(torch.atleast_2d, np.atleast_2d, device, dtype)
         self._test_atleast_dim(torch.atleast_3d, np.atleast_3d, device, dtype)
 
+    @onlyOnCPUAndCUDA
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @dtypes(*(torch.testing.get_all_int_dtypes() + torch.testing.get_all_fp_dtypes(include_bfloat16=False)))
+    def test_argminmax_multiple(self, device, dtype):
+        # Reference: https://github.com/pytorch/pytorch/issues/41998
+        t = torch.ones(3, 3, device=device, dtype=dtype)
```

Comment:
This test needs to be expanded. It can have fixtures (tensors you've designed), but if it does they should be in an iterable you enumerate over. It should also generate random integer tensors of various sizes and dimensionality, and test that the function is like NumPy not only on them but also their transposes (to test discontiguous tensors). The test can also programmatically enumerate the dim arguments to test each dim.

Reply:
Sure will do that. I thought since the PR is fixing a behaviour, minimal test verifying it should be fine.